src/DSharp.Mscorlib/Array.cs

```diff
@@ -8,23 +8,27 @@ namespace System
     [ScriptIgnoreNamespace]
     [ScriptImport]
     [ScriptName("Array")]
-    public sealed class Array : ICollection
+    public sealed class Array : ICollection // Array implents  ICloneable, IList, IStructuralComparable, IStructuralEquatable on CLR
     {
         [ScriptField]
         [ScriptName("length")]
         public extern int Length { get; }
 
         [ScriptField]
         [ScriptName("length")]
+        // This is not on CLR
         public extern int Count { get; }
 
         [ScriptField]
         public extern object this[int index] { get; set; }
 
+        // This is not on CLR
         public extern Array Concat(params object[] objects);
 
+        // This is not on CLR
```

Comment:
We do have one From IList in .Net 4.7.1 -> https://referencesource.microsoft.com/#mscorlib/system/array.cs,156e066ecc4ccedf
It's kinda hideous though :D